cordova-common/src/superspawn.js

```diff
@@ -113,11 +139,15 @@ exports.spawn = function(cmd, args, opts) {
         child.stdout.setEncoding('utf8');
         child.stdout.on('data', function(data) {
             capturedOut += data;
+            d.notify({'stdout': data});
```

Comment:
`d` is a `Deferred` object. See https://github.com/apache/cordova-lib/blob/master/cordova-common/src/superspawn.js#L68